File: back/src/main/java/csh/back/domain/vote/item/dto/response/VoteItemSaveResponseDto.java
package csh.back.domain.vote.item.dto.response;

import csh.back.domain.vote.item.entity.VoteItem;

public record VoteItemSaveResponseDto(String place) {
    public static VoteItemSaveResponseDto from(VoteItem voteItem) {
        return new VoteItemSaveResponseDto(
                voteItem.getTripPlace().getName()
        );
    }
}

File: back/src/main/kotlin/csh/back/domain/vote/item/dto/response/VoteItemSaveResponseDto.kt
package csh.back.domain.vote.item.dto.response

import csh.back.domain.vote.item.entity.VoteItem

data class VoteItemSaveResponseDto(val place: String) {
    companion object {
        @JvmStatic
        fun from(voteItem: VoteItem): VoteItemSaveResponseDto =
            VoteItemSaveResponseDto(voteItem.tripPlace.name)
    }
}